genie-biogem/src/fortran/biogem_box.f90

```diff
@@ -5045,6 +5036,7 @@ function fun_audit_combinetracer(dum_ocn)
     ! NOTE: adjust ALK for H2S (assumed created via sulphate reduction and thus ocean ALK increase)
     ! NOTE: subtract 2.0 x NH4 from O2 potential inventory to take into account virtual O2 liberation during ammoniam oxidation:
     !       NH4+ + 2O2 -> NO3- + 2H+ + H2O
+    !       BUT ... -(3.0/4.0) balances NH4 production from PON (WHY???)
```

Comment:
This is to balance N when using standard reactions for NO3 uptake and nitrification to determine PON remineralisation as:
  ! NOTE: to balance the uptake of NO3 into organic matter
    !       [2H+ + 2NO3- -> 2PON + (5/2)O2 + H2O]
    !       with the release of N as ammonium and subsequent oxidation to NO3 ...
    !       [NH4+ + 2O2 -> NO3- + 2H+ + H2O]
    !       the remin of PON needs to be adjusted in order that everything is conserved:
    !       2PON + 3H2O + 2H+ --> 2NH4+ + (3/2)O2
    !       and per N:
    !       PON + (3/2)H2O + H+ --> NH4+ + (3/4)O2